models/experimental/yolov4/demo/demo.py

```diff
@@ -589,6 +619,31 @@ def test_yolov4_model(device, model_location_generator, reset_seeds, input_path,
     sized = cv2.resize(img, (width, height))
     sized = cv2.cvtColor(sized, cv2.COLOR_BGR2RGB)
 
+    durations = []
     for i in range(2):  # This 'for' loop is for speed check
+        start = time.time()
         # Because the first iteration is usually longer
         do_detect(ttnn_model, sized, 0.3, 0.4, n_classes, device, class_name=namesfile, imgfile=imgfile)
+        end = time.time()
```

Comment:
I could be lacking context, but not sure what we're expecting when measuring/reporting the perf here, but `do_detect` doesn't seem like it's written for performance.
There are print statements in the function, and seems like we recreate YoloLayers each call as well as reloading class names, but at least in this setup the dependencies are static. Would the dependencies here be static as well in real use case/demo?

Reply:
@mywoodstock do cnns still need a compile and optimized/cache run, before being able to correctly measure inference time? Ex. Resnet does 2 passes before we're considered in fully inference/steady state setup.

Reply:
@tt-aho the `do_detect()` includes both the ttnn model run + pre and post-processing running on CPU.
The dependencies should be static, and the `demo.py` is mainly to show functionality not perf.
For the production run, I'll optimize the do_detect() code

Reply:
@tt-aho Yes, cnns do two passes before perf run